themes/classic/templates/catalog/_partials/product-details.tpl

```diff
@@ -50,7 +50,11 @@
           <dl class="data-sheet">
             {foreach from=$product.features item=feature}
               <dt class="name">{$feature.name}</dt>
-              <dd class="value">{$feature.value}</dd>
+              {if $feature.id_feature === '4'}
```

Comment:
@aleeks it's ok to change the "weight" and other informations about product related to the stuff inserted into "Combinations" tab but "Features" are completely different part of the system.